lib/game/game.dart

```diff
@@ -51,14 +51,21 @@ abstract class Game {
   /// Use for caluclating the FPS.
   void onTimingsCallback(List<FrameTiming> timings) {}
 
-  /// Returns the game widget. Put this in your structure to start rendering and updating the game.
-  /// You can add it directly to the runApp method or inside your widget structure (if you use vanilla screens and widgets).
-  Widget get widget => builder.build(this);
-
   void _handleKeyEvent(RawKeyEvent e) {
     (this as KeyboardEvents).onKeyEvent(e);
   }
 
+  void attach(PipelineOwner owner, BuildContext context) {
```

Comment:
Why not just use onAttach?

Reply:
Attach is the action. Prefixing with `on` normally refers to the reactions, which are triggered internally.

Reply:
Mind that onAttach still exists.

Reply:
I am not sure with both needs to exists, `attach` calls `onAttach`, I think since `onAttach` has `@mustCallsuper` it is fine to be with just that method.

But I have no strong opinion on this, so I leave it up to you

Reply:
An API with only `onAttach` would be too much error-prone. What if the user does stuff before calling super? What is the point of passing the build context and pipeline owner to a user-provided callback?

Reply:
Like I said, it is up to you 😉